staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -716,10 +716,10 @@ func (a *Admission) prioritizePods(pods []*corev1.Pod) []*corev1.Pod {
 }
 
 func containsString(needle string, haystack []string) bool {
-	for _, s := range haystack {
-		if s == needle {
-			return true
-		}
+	sort.Strings(haystack)
```

Comment:
This is O(n*log(n)) in average case so it will be slower than just going through strings which is O(n). So overall operation will be slower. Currently it is O(n) and after your change it will be O(n*log(n) + log(n)).

Reply:
ok. I get it,thank you.